torchok/tasks/base.py

```diff
@@ -121,17 +122,18 @@ def on_predict_start(self) -> None:
         if self._hparams.task.load_checkpoint is not None:
             load_checkpoint(self, **self._hparams.task.load_checkpoint)
 
-    def training_step(self, batch: Dict[str, Union[torch.Tensor, int]], batch_idx: int) -> Dict[str, torch.Tensor]:
+    def training_step(self, batch: Dict[str, Union[Tensor, int]], batch_idx: int) -> Dict[str, Tensor]:
         """Complete training loop."""
         output = self.forward_with_gt(batch)
         total_loss, tagged_loss_values = self.losses(**output)
+        self.log("loss", total_loss, prog_bar=True, on_step=True)
```

Comment:
We don't usually log on each step. Why did you decide it to do for user?

Reply:
But we didn't track loss on each single step. It should be tracked with the configured frequency. So, how do configure that?

Reply:
Your CMD logs included step loss. The step size is determined in the config in TQDMProgressBar callback. Loss was tracked automatically in the previous version, but not it is not.

Reply:
Because I want loss to be displayed at every step